test/integration/framework/src/siftool/sifchain.py

```diff
@@ -201,6 +211,111 @@ def wait_up(self, host, port):
                 time.sleep(1)
 
 
+# Refactoring in progress
```

Comment:
Is this comment still valid?

Reply:
Yes, it's actually new :-) Still lots of rough edges everywhere.